site/docs/latest/deployment/manual.md

```diff
@@ -23,31 +23,33 @@ zkServers=100.0.0.1:2181,100.0.0.2:2181,100.0.0.3:2181
 
 > A full listing of configurable parameters available in `bookkeeper-server/conf/bk_server.conf` can be found in the [Configuration](../../reference/config) reference manual.
 
-Once the bookie's configuration is set, you can start it up using the [`bookie`](../../reference/cli#bookkeeper-bookie) command of the [`bookkeeper`](../../reference/cli#bookkeeper) CLI tool:
+Once the bookie's configuration is set, you can set up cluster metadata for the cluster by running the following command from any bookie in the cluster:
 
 ```shell
-$ bin/bookkeeper bookie
+$ bookkeeper-server/bin/bookkeeper shell metaformat
```

Comment:
Can you please use the new 'bkctl' utility?
Then the command is 'initnewcluster'

Reply:
@eolivelli I tried `bkctl`, but there is no `initnewcluster` command, only `bin/bkctl cluster init`, and need to configure the `service URI`. In my tests only `bin/bookkeeper shell` supports `initnewcluster`.